scripts/sct_compute_mtr.py

```diff
@@ -105,18 +57,26 @@ def get_parser():
         help='Path to output file.',
         metavar=Metavar.str,
         default=os.path.join('.','mtr.nii.gz'))
-
     return parser
 
 
-#=======================================================================================================================
-# Start program
-#=======================================================================================================================
+def main():
+    # Check input parameters
+    parser = get_parser()
+    args = parser.parse_args(args=None if sys.argv[1:] else ['--help'])
+    fname_mtr = args.o
+    verbose = args.v
+    # Update log level
+    sct.init_sct(log_level=verbose, update=True)
+
+    # compute MTR
+    sct.printv('\nCompute MTR...', verbose)
```

Comment:
Should we leave this with double quotes as stated in the [proposed convention](https://github.com/neuropoly/spinalcordtoolbox/issues/2439#issuecomment-531304020) or should we wait until we fully agree on it?

Reply:
let's wait, and also try to minimize the number of cosmetic changes unrelated to this PR (although i did some...)